### PR TITLE
Env docker

### PR DESCRIPTION
**Pull request by PetN5026**
#80 
Added lines to dockerfile to .env into docker container.
FE will need to have a .env at root with the two variables. BE can provide the file as well.
ALGO
SECRET_SECURITY_KEY

### CODE REVIEW

File: server/queries/users.py [hunk 1]
```diff
@@ -204,13 +204,16 @@ def db_get_user_by_username(
     db: Session,
     username: str,
 ):
-    filtered_username = username.strip()
+    filtered_username = username.strip().lower()
+    username = username.lower()
     user_username = (
         db.query(User)
-        .filter(User.username.ilike(f"%{filtered_username}%"))
+        # .filter(User.username.ilike(f"%{filtered_username}%"))
```
Comment (arturparkhisenko) [inline server/queries/users.py:211]: @PetN5026 you can remove this comment. And please format the next line.

File: server/queries/users.py [hunk 2]
```diff
@@ -204,13 +204,16 @@ def db_get_user_by_username(
...
+        .filter(func.lower(User.username) ==filtered_username)
```
Comment (arturparkhisenko) [inline server/queries/users.py:212]: Do we need to do this `func.lower(User.username)` when we have `username = username.lower()` or they're different?

Comment (arturparkhisenko) [reply server/queries/users.py:212]: cc @PetN5026 ^

Comment (PetN5026) [reply server/queries/users.py:212]: It's different, the func is for sqlalchemy to turn the name in the db to lowercase.

File: server/queries/users.py (hunk 1 above)
Comment (PetN5026) [reply server/queries/users.py]: Deleted and formatted in a new commit.

**Review by arturparkhisenko — APPROVED** [review]

**Pull request merged**